resip/recon/ConversationManager.hxx

```diff
@@ -483,6 +486,18 @@ public:
    virtual void onDtmfEvent(ParticipantHandle partHandle, int dtmf, int duration, bool up) = 0;
 
    ///////////////////////////////////////////////////////////////////////
+   // Pager Message Handlers /////////////////////////////////////////////
+   ///////////////////////////////////////////////////////////////////////
+
+   virtual void onMessageArrived(resip::ServerPagerMessageHandle handle, const resip::SipMessage& message) = 0;
+   // Called when a MESSAGE has been successfully sent
+   virtual void onSuccess(resip::ClientPagerMessageHandle handle, const resip::SipMessage& status) = 0;
+   // Application could re-page the failed contents or just ingore it.
+   virtual void onFailure(resip::ClientPagerMessageHandle handle, const resip::SipMessage& status, std::auto_ptr<resip::Contents> contents) = 0;
+   
+   const char* sendMessage(const resip::NameAddr& destination, const resip::Data& msg, const resip::Mime& mimeType);
```

Comment:
this is a method that may belong in UserAgent instead of ConversationManager because MESSAGE doesn't create a dialog.  It is discussed in RFC 3428 section 2
https://tools.ietf.org/html/rfc3428#section-2